frontend/pages/index.tsx

```diff
@@ -48,228 +47,265 @@ type HomeProps = {
-                    <span className="space-y-2 text-black text-xl whitespace-nowrap">
-                    Sign up for the RSD
-                    </span>
-                </div>
+  // Initialize AOS library
+  useEffect(() => {
+    AOS.init()
+  }, [])
 
-              </button>
+  const handleClickOpen = () => {
+    const loginButton = document.querySelector('.rsd-login-button')
+    if (loginButton) {
+      const evt = new MouseEvent('click', {
+        bubbles: true
+      })
+      loginButton.dispatchEvent(evt)
+    }
+  }
 
-            </div>*/}
-            <div className="flex gap-4 md:gap-10 mt-10 items-center">
-              <Link href="/software" passHref>
-                <a className="flex gap-4 cursor-pointer"
-                   data-aos="fade" data-aos-offset="200" data-aos-delay="50"
-                   data-aos-duration="1000"
-                   tabIndex={0}>
-                  <div className="relative group">
-                    <div
-                      className="absolute -inset-1 bg-gradient-to-r from-purple-600 to-pink-600 rounded-lg blur opacity-25 group-hover:opacity-100 transition duration-1000 group-hover:duration-300"/>
-                    <div
-                      className="flex gap-3 text-black relative px-8 py-3 bg-white ring-1 ring-gray-900/5 rounded leading-none items-center space-x-2 ">
-                    <span className="space-y-2 text-xl font-medium  whitespace-nowrap ">
-                      Discover Software
-                    </span>
-                    </div>
-                  </div>
-                </a>
-              </Link>
+  return (
+     <div className="bg-white">
+
+        <AppHeader/>
 
-              <a data-aos="fade" data-aos-offset="200" data-aos-delay="150" data-aos-duration="1000"
-                 className="text-xl leading-6 text-transparent bg-clip-text bg-gradient-to-br from-[#FD54BB] to-[#1BECCB] "
-                 target="_blank"
-                 rel="noreferrer"
-                 href="mailto:rsd@esciencecenter.nl?subject=More information about the RSD&body=Hi,
-                  I would like to get in contact to start my organization within the Research
-                  Software Directory"
-              >
-                Register your <br/> Organisation
+        {/* Head and claim */}
+        <div className="bg-secondary bg-landing-page">
+          <div className="flex flex-row flex-wrap container mx-auto px-6 md:px-10 pt-16 pb-12 max-w-screen-xl text-white">
+            <div className="min-w-min flex flex-col">
+              <LogoHelmholtz width="220" />
+              <div className="pt-2 pb-12">Spitzenforschung für<br />große Herausforderungen.</div>
+              <a onClick={handleClickOpen}>
+                <div className="w-[250px] bg-[#05e5ba] hover:bg-primary text-secondary hover:text-white text-center font-medium text-2xl py-4 px-6 rounded-sm">
+                  Add your software
```

Comment:
Noticed during testing: Prior authentication this button finally redirects to the Software overview. After being authenticated the button does nothing.